src/applications/edu-benefits/1995/config/new-chapters.js

```diff
@@ -50,29 +55,14 @@ export const newChapters = {
       benefitSelection: {
         title: 'Education benefit',
         path: 'benefits/eligibility',
-        uiSchema: {
-          benefit: {
-            'ui:widget': 'radio',
-            'ui:title': 'Which benefit are you currently using?',
-            'ui:options': {
-              labels: benefitsLabels,
-            },
-          },
-        },
-        schema: {
-          type: 'object',
-          properties: {
-            benefit,
-          },
-        },
+        uiSchema: benefitSelection.uiSchema,
+        schema: benefitSelection.schema,
       },
       // related to 1995-STEM
       stem: {
         title: 'Education benefit',
         path: 'benefits/stem',
-        depends: {
-          benefit: 'chapter33',
-        },
+        depends: form => displayEdithNourseRogersScholarship(form),
```

Comment:
This can just be `displayEdithNourseRogersScholarship`

Reply:
with the removal of the schema modification this check should be able to go back to the way it was